native/machine-coding/bill-spill/style.css

```diff
@@ -0,0 +1,205 @@
+* {
```

Comment:
please import the style as done for other project styles

Reply:
done